specification/cognitiveservices/data-plane/Speech/BatchTextToSpeech/stable/2024-04-01/batchsynthesis.json

```diff
@@ -0,0 +1,779 @@
+          "format": "int64",
+          "description": "The number of billable characters for personal voices in the input file.",
+          "type": "integer"
+        }
+      },
+      "readOnly": true
+    },
+    "BatchSynthesisConfig": {
+      "description": "Text-to-speech configuration for plain text input.",
+      "required": [
+        "voice"
+      ],
+      "type": "object",
+      "properties": {
+        "voice": {
+          "description": "Voice name for PlainText input. You can find platform voice list here https://learn.microsoft.com/en-us/azure/ai-services/speech-service/language-support?tabs=tts.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "style": {
+          "description": "Style name for PlainText input.",
+          "type": "string"
+        },
+        "rate": {
+          "description": "Speaking rate for PlainText input. For supported values, see https://learn.microsoft.com/en-us/azure/ai-services/speech-service/speech-synthesis-markup-voice#adjust-prosody.",
+          "type": "string"
+        },
+        "pitch": {
+          "description": "Pitch for PlainText input. For supported values, see https://learn.microsoft.com/en-us/azure/ai-services/speech-service/speech-synthesis-markup-voice#adjust-prosody.",
+          "type": "string"
+        },
+        "volume": {
+          "description": "Volume for PlainText input. For supported values, see https://learn.microsoft.com/en-us/azure/ai-services/speech-service/speech-synthesis-markup-voice#adjust-prosody.",
+          "type": "string"
+        },
+        "backgroundAudio": {
+          "$ref": "#/definitions/BatchSynthesisBackgroundAudioDefinition"
+        }
+      }
+    },
+    "BatchSynthesisError": {
+      "description": "Batch Synthesis Error.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Top level error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message",
+          "type": "string"
+        }
+      },
+      "readOnly": true
+    },
+    "BatchSynthesisInput": {
+      "description": "Batch Synthesis input.",
+      "required": [
+        "content"
+      ],
+      "type": "object",
+      "properties": {
+        "content": {
+          "description": "It could be SSML or plain text.",
+          "minLength": 1,
+          "type": "string"
+        }
+      }
+    },
+    "BatchSynthesisOutputs": {
+      "description": "Batch Synthesis output.",
+      "type": "object",
+      "properties": {
+        "result": {
+          "description": "sas url of the result file. It would be relative path when \"destinationContainerUrl\" is specified.",
+          "type": "string"
+        },
+        "summary": {
+          "description": "When BatchSynthesisProperties.decompressOutputFiles is true, job summary file path will be provided here.",
+          "type": "string"
+        }
+      },
+      "readOnly": true
```

Comment:
The `readOnly` attribute should only be specified in schema property definitions.

Reply:
Thank you and fixed.